spring-cloud-gcp-data-spanner/pom.xml

```diff
@@ -1,47 +1,59 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xmlns="http://maven.apache.org/POM/4.0.0"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-	<modelVersion>4.0.0</modelVersion>
```

Comment:
@alexkhimich Thanks for reverting the formatting changes! This file is one you seem to have missed. Yeah I know tabs vs spaces. :smiley:

Reply:
Also I've pointed that I would like do not see Powermock dependency.
Please, follow standard Spring Boot set of Testing frameworks.